org.eclipse.wildwebdeveloper/src/org/eclipse/wildwebdeveloper/debug/AbstractRunHTMLDebugTab.java

```diff
@@ -148,6 +190,18 @@ public void initializeFrom(ILaunchConfiguration configuration) {
 			this.argumentsText.setText(configuration.getAttribute(AbstractHTMLDebugDelegate.ARGUMENTS, "")); //$NON-NLS-1$
 			this.workingDirectoryText.setText(
 					configuration.getAttribute(AbstractHTMLDebugDelegate.CWD, pathOrEmpty(getSelectedProject())));
+			this.urlText.setText(configuration.getAttribute(ChromeRunDAPDebugDelegate.URL, "")); //$NON-NLS-1$
+			boolean urlDebugMode = configuration.getAttribute(AbstractHTMLDebugDelegate.URL_DEBUG, false);
```

Comment:
I don't think it's worth storing the file if URL is used or vice-versa.
IMO, we can get rid of the URL_DEBUG attribute and just check `file == null` to know whether it's a URL or a File Launch.